paver.sh

```diff
@@ -98,7 +98,13 @@ clean_build() {
     fi
     # In some case pip hangs with a build folder in temp and
     # will not continue until it is manually removed.
-    rm -rf /tmp/pip*
+    # On the OSX build server tmp is in $TMPDIR
+    if [ ! -z "${TMPDIR-}" ]; then
```

Comment:
`-n`  works. 

the parameter expansion is used because if we test only for `${TMPDIR}` we might get 'unbound variable' error when it is not defined.

thanks!